rvt/vis.py

```diff
@@ -158,12 +158,17 @@ def slope_aspect(dem,
     if resolution_x < 0 or resolution_y < 0:
         raise Exception("rvt.visualization.slope_aspect: resolution must be a positive number!")
 
-    # change no_data to np.nan
-    if no_data is not None:
+    # Make sure array has the correct dtype!
+    dem = dem.astype(np.float32)
+
+    # Change no_data to np.nan
+    if no_data and ~np.isnan(no_data):
```

Comment:
This won't work in case no_data=0 😃

Reply:
I would also keep existing functionality 😄. Could be confusing if user specifies nodata value and we don't do anything because image also contains nan.